### PR TITLE
Add magicFirewallSamplesAdaptiveGroups graphql node

### PR DESCRIPTION
**Pull request by tejaswi-cf**
* Add details about the magicFirewallSamplesAdaptiveGroups node
  to the analytics docs.

### CODE REVIEW

**Review by tlozoot — APPROVED** [review]

**Comment by tejaswi-cf** [comment]
Please don't merge the changes till we have completed validating the corresponding code changes on our side.
I raised a PR so I can get some early feedback.

**Review by player-two — APPROVED** [review]

File: products/analytics/src/content/graphql-api/limits/index.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Access to Network Analytics <b>ipFlows</b> nodes is only available to Enterprise
 | `loadBalancingRequestsGroups`        |      n/a |   3 days |  30 days |    30 days |
 | `loadBalancingRequests`              |      n/a |   3 days |  30 days |    30 days |
 | `synAvgPps1mGroups`                  |      n/a |      n/a |      n/a |     7 days |
+| `magicFirewallSamplesAdaptiveGroups` |      n/a |      n/a |      n/a |     7 days |
```
Comment (nadinelyab) [inline products/analytics/src/content/graphql-api/limits/index.md:44]: Our data retention is 30 days so I believe that's what this value should be too.

Comment (tejaswi-cf) [reply products/analytics/src/content/graphql-api/limits/index.md:44]: This field is for accessing the data. While defining the schema we have used the default value of notOlderThan, so queries can only access data from the past 7 days.

Comment (nadinelyab) [reply products/analytics/src/content/graphql-api/limits/index.md:44]: Got it. Looks good then

**Review by nadinelyab — APPROVED** [review]

**Comment by tejaswi-cf** [comment]
Thanks for reviewing the changes. Merging them now.

**Pull request merged**